knowledge-base/check-if-data-field-exists-before-using-it.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Check If Data Field Exists Before Using It
+description: How to avoid Report error when data field does not exist
+type: how-to
+page_title: Avoid Report error when no such data field
+slug: check-if-data-field-exists-before-using-it
+position: 
+tags: 
+ticketid: 1462533
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+In some scenarios, part of the fields used to design a report may not be available runtime. This will cause an exception in 
+the report stating that the corresponding field is not defined in the current scope.
```

Comment:
I would add one more sentence explaining how this error will be displayed (red rectangle with some text) and why it would be aproblem (may generate more pages)